File: python/examples/gemini_generate.py
from buble_ai import Buble

client = Buble()

response = client.chat.gemini.generate_content(
    "openai/gpt-5.5",
    contents=[
        {
            "role": "user",
            "parts": [{"text": "Write a short launch summary."}],
        }
    ],
)

print(response["candidates"][0]["content"]["parts"][0]["text"])

File: php/examples/gemini-generate.php
<?php

declare(strict_types=1);

require __DIR__ . '/../vendor/autoload.php';

use Buble\BubleClient;

$client = BubleClient::fromEnv();

$response = $client->chat()->gemini()->generateContent('openai/gpt-5.5', [
    'contents' => [
        [
            'role' => 'user',
            'parts' => [
                ['text' => 'Write a short launch summary.'],
            ],
        ],
    ],
]);

echo $response['candidates'][0]['content']['parts'][0]['text'] . PHP_EOL;
